package.json

```diff
@@ -90,6 +90,8 @@
     "knitwork": "^1.0.0",
     "magic-string": "^0.30.4",
     "mlly": "^1.4.2",
+    "nuxt-site-config": "^1.4.0",
```

Comment:
@harlan-zw https://nuxtseo.com/site-config/getting-started/installation specifies that `nuxt-site-config` has to be installed along with `nuxt-site-config-kit`. As I only ever made use of a `nuxt-site-config-kit` import in this PR and ignoring the playground's `useSiteConfig` usage right now, I wonder: is it really required to install `nuxt-site-config` as well? And if so, why?